Extension/package.json

```diff
@@ -2361,6 +2361,38 @@
           "description": "%c_cpp.configuration.doxygen.generatedStyle.description%",
           "scope": "resource"
         },
+        "C_Cpp.doxygen.selectDoxygenTags": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "attention",
+              "author",
+              "authors",
+              "bug",
+              "copyright",
+              "date",
+              "details",
+              "exception",
+              "invariant",
+              "note",
+              "pre",
+              "post",
+              "remark",
+              "remarks",
+              "result",
+              "return",
+              "since",
+              "test",
+              "todo",
+              "version",
+              "warning"
+              ]
+            },
+            "default": {},
+            "description": "%c_cpp.configuration.doxygen.selectDoxygenTags.description%",
+            "scope": "resource"
```

Comment:
If this is a "resource" scope, would it make sense to also change c_cpp.configuration.simplifyStructuredComments to resource since they're related?